crates/proof-of-sql/src/base/arrow/owned_and_arrow_conversions.rs

```diff
@@ -105,6 +105,10 @@ impl<S: Scalar> From<OwnedColumn<S>> for ArrayRef {
             }
             OwnedColumn::Scalar(_) => unimplemented!("Cannot convert Scalar type to arrow type"),
             OwnedColumn::VarChar(col) => Arc::new(StringArray::from(col)),
+            OwnedColumn::VarBinary(col) => {
+                let arr = BinaryArray::from_iter_values(col.iter().map(std::vec::Vec::as_slice));
+                Arc::new(arr)
```

Comment:
Don't use `std`.
Instead add `use alloc::vec::Vec`.

NIT:
```suggestion
                Arc::new(BinaryArray::from_iter_values(col.iter().map(Vec::as_slice)))
```